src/test/ui/error-codes/E0658.stderr

```diff
@@ -4,7 +4,7 @@ error[E0658]: use of unstable library feature 'i128' (see issue #35118)
 LL |     let _ = ::std::u128::MAX; //~ ERROR E0658
    |             ^^^^^^^^^^^^^^^^
    |
-   = help: add #![feature(i128)] to the crate attributes to enable
+   = help: add #![feature(repri128)] to the crate attributes to enable
```

Comment:
s/repri128/repr128

Reply:
Er, shouldn't this test be testing another feature now :P

Reply:
Yeah, the whole stderr needs to change... I will fix this one after I get the rest compiling successfully.